server/database/build.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+DROP TABLE IF EXISTS users, tickets_status, tickets CASCADE;
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    email TEXT NOT NULL UNIQUE,
+    name VARCHAR NOT NULL ,
+    password VARCHAR NOT NULL
+);
+CREATE TABLE tickets_status(
+    uid SERIAL PRIMARY KEY,
+    status VARCHAR NOT NULL
+);
+CREATE TABLE tickets(
+    uid SERIAL PRIMARY KEY,
+    from_tickets VARCHAR NOT NULL,
+    to_tickets VARCHAR NOT NULL,
+    date INTEGER ,
```

Comment:
I don't recomment using INTEGER for date type. SQL has support for the following types:

DATE | Stores date in the format YYYY-MM-DD
-- | --
TIME | Stores time in the format HH:MI:SS
DATETIME | Stores date and time information in the format YYYY-MM-DD HH:MI:SS
TIMESTAMP | Stores number of seconds passed since the Unix epoch (‘1970-01-01 00:00:00’ UTC)
YEAR | Stores year in 2 digit or 4 digit format. Range 1901 to 2155 in 4-digit format. Range 70 to 69, representing 1970 to 2069.

so I suggest we use one of them.
A timestamp is good to have on all tables that interact with input from the user for archiving and logging purposes, so it's good to include that too.
I suggest we add a timestamp column to the table, and to use ```DATE``` or ```DATETIME``` for the date column.